3-Network-Configuration/3.4-Uncommon-Network-Protocols.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats
+
+@test "3.4.1 Ensure DCCP is disabled (Automated)" {
+    run bash -c "modprobe -n -v dccp"
+    [ "$status" -eq 0 ]
+    [ "$output" = "install /bin/true " ]
+    run bash -c "lsmod | grep dccp"
+    [ "$status" -ne 0 ]
+    [ "$output" = "" ]
+}
+
+@test "3.4.2 Ensure SCTP is disabled (Automated)" {
+    run bash -c "modprobe -n -v sctp"
```

Comment:
Please change this to the command listed in the CIS document (`modprobe -n -v sctp | grep -E '(sctp|install)'`)